mautic/mautic
# Remove and adding options to select and multiselect fields removes some options from existing saved options

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description
Remove existing options and then adding options to select and multiselect fields removes some options from existing saved options.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new select type custom field with 3 options
3. Save it and edit again
4. Remove 2nd option and add new option
5. save it
6. Now all available options should be correct.

Same issue is exist on multiple places :-
1. Select / Multiselect field in Custom object field
2. Select / Multiselect field in form field
